src/common/types/objects/trustlines.ts

```diff
@@ -18,7 +18,7 @@ export interface Trustline {
 
 export type FormattedTrustlineSpecification = {
```

Comment:
This type should not be necessary, since you can just directly use the account_lines params